test/devices_tests/fan_test.py

```diff
@@ -60,6 +60,73 @@ async def test_sync_state_address(self):
         )
 
     #
+    # TEST SWITCH ON/OFF
+    #
+    async def test_switch_on_off(self):
+        """Test switching on/off of a Fan."""
+        xknx = XKNX()
+        fan = Fan(xknx, name="TestFan", group_address_speed="1/2/3")
+
+        # Turn the fan on via speed GA. First try without providing a speed,
+        # which will not work for fans without a switch GA.
+        await fan.turn_on()
+        assert xknx.telegrams.qsize() == 0
```

Comment:
Should we just use a default value (eg. 50%) then?

Reply:
That is what the KNX HA integration does, but I left that logic there rather then moving into xknx, to not break the xknx behavior. Although having it in xknx would have one big benefit:

Currently, if you have a fan with a switch GA and `turn_on()` is called, we have to ignore the provided speed percentage value, since we do not know whether it is the default or a dedicated percentage value. If xknx were to handle it the default logic, we could support the percentage value also for switch GA driven fans, since xknx would be able to differentiate between a real percentage value (sent by the user or an HA automation) and just the default (with the later being used for purely speed GA fans, but not for switch GA ones).

Reply:
Move it to xknx then! 
It's ok to break things here if we get benefits from it (even if just for readability).

Reply:
Done